data/human/rim jobs.txt

```diff
@@ -111,6 +119,12 @@ mission "Han Sizer Month [1]"
 	repeat
 	description `In celebration of Han Sizer month, bring <fare> to all the marked systems and return to <destination> for <payment>.`
 	passengers 2 10 .8
+	substitutions
+		"<on Longjump>" "someone near here."
```

Comment:
This will resolve, lower down there, to `One is already uploading a travelogue to someone someone near here. as they start walking away.`

Reply:
Thanks! Removed the extra "someone".